staging/src/k8s.io/kube-openapi/pkg/aggregator/aggregator.go

```diff
@@ -320,6 +357,8 @@ func equalIntPointer(s1, s2 *int64) bool {
 
 // EqualSchema returns true if models have the same properties and references
 // even if they have different documentation.
+// Note: Dependencies and Enum are not supported. EqualSchema will return
```

Comment:
I guess we have a limited number of a types for enums. It's just odd to have this logic in an equality implementation and not in a validation somewhere else.